engine/src/main/resources/assets/i18n/menu_fa.lang

```diff
@@ -0,0 +1,244 @@
+    "could-not-connect-to-server": "نمی تواند به کارساز ( سرور) وصل شود",
+    "credits": "اعتبارها",
+    "create-game": "خلق",
+    "create-game-title": "خلق بازی",
+    "create-preview": "پیش دید کردن",
+    "custom-servers": "بر حسب عادت",
+    "deactivate-module": "غیر فعال",
+    "death-screen-title": "شما مرده اید",
+    "default-binding": "از بین رفتن اتصال",
+    "delete-game": "حذف کردن",
+    "dev-tools-bt-editor": "نمایشگر درختی ویرایش رفتاری",
+    "dev-tools-nui-editor": "ویرایشگر صفحه ان. یو. ال",
+    "dev-tools-skin-editor": "ویرایشگر اسکین ان.یو. ال",
+    "dev-tools-title": "ابزار توسعه دهنده",
+    "dialog-cancel": "لغو کردن",
+    "dialog-ok": "درست است",
+    "disable-all-modules": "از کار انداختن همه",
+    "download-module": "دانلود",
+    "downloading-server-list": "دانلود لیست کارساز..",
+    "drop-item": "قطع مطلب",
+    "edit-server": "ویرایش",
+    "edit-server-title": "اضافه/ویرایش  کارساز (سرور)",
+    "enter-username-message": "Hello and welcome to Terasology's multiplayer!\nWould you like to use a non-default username?",
+    "enum-editor-prompt": "انتخاب مقدار برای منحنی",
+    "enum-editor-title": "ویرایشگر منحنی داده ها",
+    "error-downloading-server-list": "اشتباه در دانلود فهرست کترساز (سرور)",
+    "exit-game": "خروج از بازی تراسولوژی",
+    "extra-lighting": "پرتوهای اضافه",
+    "eye-adaption": "انطباق چشمی",
+    "failed-to-join": "عدم موفقیت در پیوستن",
+    "flickering-lights": "نورهای لرزان",
+    "fog": "مه",
+    "fov-amount": "میزان میدان دید",
+    "framerate-limit": "محدودیت سازه",
+    "game-name": "اسم بازی",
+    "game-settings": "تنظیمات",
+    "game-worlds": "دنیاها",
+    "gameplay": "محتویات نمایش بازی",
+    "gameplay-info": "can click 'modules' to customize further",
+    "host-multiplayer-game": "بازی میزبان",
+    "ingame-menu-title": "منوی در حین بازی",
+    "input-settings": "ورودی",
+    "input-settings-title": "تنظیمات ورودی",
+    "invert-mouse": "موشواره وارونه",
+    "invert-x": "معکوس ایکس اکسیس",
+    "invert-y": "معکوس وای اکسیس",
+    "invert-z": "معکوس زد اکسیس",
+    "join-game-online": "پیوستن به بازی",
+    "join-server-refresh": "تازه کردن",
+    "join-server-action": "پیوستن",
+    "join-server-requested": "تقاضا شده",
+    "join-server-title": "اتصال به سرور",
+    "light-shafts": "پرتوهای نور",
+    "listed-servers": "لیست شده",
+    "load-game": "ذخیره گذاری",
+    "menu-animations": "فهرست فیلمهای نقاشی شده",
+    "missing-name-message": "شما باید اسم را وارد کنید",
+    "module-dependencies-empty": "This module has no dependencies",
+    "module-dependencies-exist": "توابع طرحها",
+    "module-filter": "فیلتر",
+    "module-list": "طرح ها",
+    "module-name": "اسم",
+    "module-status": "وضعیت",
+    "module-status-activated": "فعال شده",
+    "module-status-activegameplay": "فعال کردن گیم پلی",
+    "module-status-available": "در دسترس",
+    "module-status-dependency": "تبعیت",
+    "module-status-error": "Incompatible or unresolved dependencies",
```

Comment:
This is untranslated - can you wrap this up just so the language is 100% finished? [7]